### PR TITLE
Updates priorities in cs-rules.xml.

### PR DESCRIPTION
**Pull request by pbruneton**
All rules were set to MAJOR.
Use native fxcop severities instead.

### CODE REVIEW

**Comment by dbolkensteyn** [comment]
Thanks @pbruneton , this seems a good idea!

**Comment by brugidou** [comment]
Hi, any news on this? Could we get it merged?

File: src/main/resources/org/sonar/plugins/fxcop/cs-rules.xml [hunk 1]
```diff
@@ -119,7 +119,7 @@ Generally, both of the prior declarations should be avoided so that the type arg
 
   <rule key="TypesThatOwnDisposableFieldsShouldBeDisposable">
     <configKey>CA1001</configKey>
-    <priority>MAJOR</priority>
+    <priority>CRITICAL</priority>
```
Comment (dbolkensteyn) [inline src/main/resources/org/sonar/plugins/fxcop/cs-rules.xml]: i'm not convinced this should be critical

File: src/main/resources/org/sonar/plugins/fxcop/cs-rules.xml [hunk 2]
```diff
@@ -514,7 +514,7 @@ Generally, both of the prior declarations should be avoided so that the type arg
 
   <rule key="UseGenericsWhereAppropriate">
     <configKey>CA1007</configKey>
-    <priority>MAJOR</priority>
+    <priority>CRITICAL</priority>
```
Comment (dbolkensteyn) [inline src/main/resources/org/sonar/plugins/fxcop/cs-rules.xml]: not conviced this should be critical

**Comment by dbolkensteyn** [comment]
I've created the following ticket to track this PR: https://jira.sonarsource.com/browse/SONARFXCOP-49

**Comment by dbolkensteyn** [comment]
This PR is now superceded by https://github.com/SonarSource/sonar-fxcop-library/pull/15 - therefore I am closing it

**Pull request closed without merging**